posts/2021-02-15-eclipse-transformer.adoc

```diff
@@ -0,0 +1,506 @@
+
+.Java EE to Jakarta EE 9 Feature Updates: Version Only
+|===
+| Java EE 8 feature name | Jakarta EE 9 feature name
+
+| appClientSupport-1.0
+| appClientSupport-2.0
+
+| appSecurity-3.0
+| appSecurity-4.0
+
+| batch-1.0
+| batch-2.0
+
+| batchManagement-1.0
+| batchManagement-2.0
+
+| beanValidation-2.0
+| beanValidation-3.0
+
+| cdi-2.0
+| cdi-3.0
+
+| concurrent-1.0
+| concurrent-2.0
+
+| javaee-8.0
+| jakartaee-9.0
+
+| javaeeClient-8.0
+| jakartaeeClient-9.0
+
+| jsonb-1.0
+| jsonb-2.0
+
+| jsonbContainer-1.0
+| jsonbContainer-2.0
+
+| jsonp-1.1
+| jsonp-2.0
+
+| jsonpContainer-1.1
+| jsonpContainer-2.0
+
+| managedBeans-1.0
+| managedBeans-2.0
+
+| mdb-3.2
+| mdb-4.0
+
+| servlet-4.0
+| servlet-5.0
+
+| webProfile-8.0
+| webProfile-9.0
+
+| websocket-1.1
+| websocket-2.0
+|===
+
+In other cases, in addition to updating the feature version, the feature short name was updated.
+
+To simplify updating server configuration files, a Jakarta EE 9 feature name which uses the new feature version while retaining the old feature name is also supported.
+
+.Java EE to Jakarta EE 9 Feature Updates: Short Name and Version
+|===
+| Java EE 8 feature name | Jakarta EE 9 feature name | Alternative Jakarta EE 9 feature name
+
+| ejb-3.2
+| enterpriseBeans-4.0
+| ejb-4.0
+
+| ejbHome-3.2
+| enterpriseBeansHome-4.0
+| ejbHome-4.0
+
+| ejbLite-3.2
+| enterpriseBeansLite-4.0
+| ejbLite-4.0
+
+| ejbPersistentTimer-3.2
+| enterpriseBeansPersistentTimer-4.0
+| ejbPersistentTimer-4.0
+
+| ejbRemote-3.2
+| enterpriseBeansRemote-4.0
+| ejbRemote-4.0
+
+| el-3.0
+| expressionLanguage-4.0
+| el-4.0
+
+| jacc-1.5
+| appAuthorization-2.0
+| jacc-2.0
+
+| jaspic-1.1
+| appAuthentication-2.0
+| jaspic-2.0
+
+| javaMail-1.6
+| mail-2.0
+| javaMail-2.0
+
+| jaxb-2.2
+| xmlBinding-3.0
+| jaxb-3.0
+
+| jaxrs-2.1
+| restfulWS-3.0
+| jaxrs-3.0
+
+| jaxrsClient-2.1
+| restfulWSClient-3.0
+| jaxrsClient-3.0
+
+| jaxws-2.2
+| xmlWS-3.0
+| jaxws-3.0
+
+| jca-1.7
+| connectors-2.0
+| jca-2.0
+
+| jcaInboundSecurity-1.0
+| connectorsInboundSecurity-2.0
+| jcaInboundSecurity-2.0
+
+| jms-2.0
+| messaging-3.0
+| jms-3.0
+
+| jpa-2.2
+| persistence-3.0
+| jpa-3.0
+
+| jpaContainer-2.x
+| persistenceContainer-3.0
+| jpaContainer-3.0
+
+| jsf-2.3
+| faces-3.0
+| jsf-3.0
+
+| jsfContainer-2.3
+| facesContainer-3.0
+| jsfContainer-3.0
+
+| jsp-2.3
+| pages-3.0
+| jsp-3.0
+
+| wasJmsClient-2.0
+| messagingClient-3.0
+| wasJmsClient-3.0
+
+| wasJmsSecurity-1.0
+| messagingSecurity-3.0
+| wasJmsSecurity-3.0
+
+| wasJmsServer-1.0
+| messagingServer-3.0
+| wasJmsServer-3.0
+|===
+
+(Source: https://github.com/OpenLiberty/open-liberty/issues/13393.)
+
+Here is the complete table, using only Java EE short names:
+
+.Java EE to Jakarta EE 9 Feature Updates: All Short Names
```

Comment:
Is this table needed?  If you keep it, it should have the same updates as above.